pkg/status/dist/templates/logsagent.tmpl

```diff
@@ -9,7 +9,24 @@ Logs-agent
   {{printDashes .name "-"}}
   {{- range .sources }}
     Type: {{ .type }}
+    {{- if .path }}
+    Path: {{ .path }}
+    {{- end }}
+    {{- if .port }}
+    Port: {{ .port }}
+    {{- end }}
+    {{- if .image }}
+    Image: {{ .image }}
+    {{- end }}
+    {{- if .label }}
+    Label: {{ .label }}
+    {{- end }}
+    {{- if .status }}
     Status: {{ .status }}
+    {{- end }}
+    {{- if .inputs }}
+    Inputs: {{ range $input := .inputs }}{{$input}} {{ end }}
```

Comment:
I think you could move this logic in the function `Get` from the status package.

Reply:
Actually, I'd like to do more in the template as I feel like these concerns don't belong in the code.

Reply:
👍